pkg/lib/cockpit-components-table.scss

```diff
@@ -97,6 +97,17 @@
         > .btn-group > .pf-c-button {
             padding: 0.25rem 0.75rem;
         }
+
+        > .pf-m-link {
+            &:hover {
+                text-decoration: underline;
```

Comment:
For underliner on hover we can use isInline property on the Button links instead of adding custom CSS.

Reply:
Oh! Yeah, that would be better.

Is there something that would handle the block expansion, or do we need custom CSS for that still?

Reply:
We need custom CSS for that.